### ISSUE TITLE
Send Alive messages more frequently than a service TTL

### ISSUE THREAD

**Issue by vasily-kirichenko** [Horusiath/Akka.Cluster.Discovery#13]
We register Consul service with some TTL, then send Alive messages with the same interval. As a result, Consul log is flooded with "Check "..." missed TTL, is now critical":
```
2018/01/23 13:47:10 [WARN] agent: Check "service:test@the-node:2552/" missed TTL, is now critical
2018/01/23 13:47:10 [INFO] agent: Synced check "service:test@the-node:2552/"
2018/01/23 13:47:10 [INFO] agent: Synced check "service:test@the-node:2552/"
2018/01/23 13:47:20 [WARN] agent: Check "service:test@the-node:2552/" missed TTL, is now critical
2018/01/23 13:47:20 [INFO] agent: Synced check "service:test@the-node:2552/"
2018/01/23 13:47:20 [INFO] agent: Synced check "service:test@the-node:2552/"
2018/01/23 13:47:30 [WARN] agent: Check "service:test@the-node:2552/" missed TTL, is now critical
2018/01/23 13:47:30 [INFO] agent: Synced check "service:test@the-node:2552/"
2018/01/23 13:47:30 [INFO] agent: Synced check "service:test@the-node:2552/"
2018/01/23 13:47:40 [WARN] agent: Check "service:test@the-node:2552/" missed TTL, is now critical
2018/01/23 13:47:40 [INFO] agent: Synced check "service:test@the-node:2552/"
2018/01/23 13:47:40 [INFO] agent: Synced check "service:test@the-node:2552/"
2018/01/23 13:48:00 [WARN] agent: Check "service:test@the-node:2552/" missed TTL, is now critical
2018/01/23 13:48:00 [INFO] agent: Synced check "service:test@the-node:2552/"
2018/01/23 13:48:00 [INFO] agent: Synced check "service:test@the-node:2552/"
2018/01/23 13:48:10 [WARN] agent: Check "service:test@the-node:2552/" missed TTL, is now critical
2018/01/23 13:48:10 [INFO] agent: Synced check "service:test@the-node:2552/"
2018/01/23 13:48:10 [INFO] agent: Synced check "service:test@the-node:2552/"
```
Would it solve the problem if we sent Alive as twice as more frequently as TTL, for example?

**Comment by Horusiath** [comment]
Valid point.

**Comment by Horusiath** [comment]
Applied and fixed as part of release v0.1.0.
